libs/core/langchain_core/prompts/image.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any
+
+from langchain_core.prompt_values import ImagePromptValue, ImageURL, PromptValue
+from langchain_core.prompts.base import BasePromptTemplate
+from langchain_core.pydantic_v1 import Field
+from langchain_core.utils import image as image_utils
+
+
+class ImagePromptTemplate(BasePromptTemplate[ImageURL]):
+    """An image prompt template for a multimodal model."""
+
+    template: dict = Field(default_factory=dict)
+    """Template for the prompt."""
+
+    def __init__(self, **kwargs: Any) -> None:
+        if "input_variables" not in kwargs:
+            kwargs["input_variables"] = []
+
+        overlap = set(kwargs["input_variables"]) & set(("url", "path", "detail"))
```

Comment:
These actually aren't required to be blocked out now that I'm using the fstring format to infer again